docs/content/en/docs/reference/clusterspec/packages.md

```diff
@@ -0,0 +1,938 @@
+      <td>string</td>
+      <td>PackageBundle</td>
+      <td>true</td>
+      </tr>
+      <tr>
+      <td><b><a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#objectmeta-v1-meta">metadata</a></b></td>
+      <td>object</td>
+      <td>Refer to the Kubernetes API documentation for the fields of the `metadata` field.</td>
+      <td>true</td>
+      </tr><tr>
+        <td><b><a href="#packagebundlespec">spec</a></b></td>
+        <td>object</td>
+        <td>
+          PackageBundleSpec defines the desired state of PackageBundle.<br/>
+        </td>
+        <td>false</td>
+      </tr><tr>
+        <td><b><a href="#packagebundlestatus">status</a></b></td>
+        <td>object</td>
+        <td>
+          PackageBundleStatus defines the observed state of PackageBundle.<br/>
+        </td>
+        <td>false</td>
+      </tr></tbody>
+</table>
+
+
+#### PackageBundle.spec
+<sup><sup>[↩ Parent](#packagebundle)</sup></sup>
+
+
+
+PackageBundleSpec defines the desired state of PackageBundle.
+
+<table>
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Type</th>
+            <th>Description</th>
+            <th>Required</th>
+        </tr>
+    </thead>
+    <tbody><tr>
+        <td><b><a href="#packagebundlespecpackagesindex">packages</a></b></td>
+        <td>[]object</td>
+        <td>
+          Packages supported by this bundle.<br/>
+        </td>
+        <td>true</td>
+      </tr></tbody>
+</table>
+
+
+#### PackageBundle.spec.packages[index]
+<sup><sup>[↩ Parent](#packagebundlespec)</sup></sup>
+
+
+
+BundlePackage specifies a package within a bundle.
+
+<table>
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Type</th>
+            <th>Description</th>
+            <th>Required</th>
+        </tr>
+    </thead>
+    <tbody><tr>
+        <td><b><a href="#packagebundlespecpackagesindexsource">source</a></b></td>
+        <td>object</td>
+        <td>
+          Source location for the package (probably a helm chart).<br/>
+        </td>
+        <td>true</td>
+      </tr><tr>
+        <td><b>name</b></td>
+        <td>string</td>
+        <td>
+          Name of the package.<br/>
+        </td>
+        <td>false</td>
+      </tr></tbody>
+</table>
+
+
+#### PackageBundle.spec.packages[index].source
+<sup><sup>[↩ Parent](#packagebundlespecpackagesindex)</sup></sup>
+
+
+
+Source location for the package (probably a helm chart).
+
+<table>
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Type</th>
+            <th>Description</th>
+            <th>Required</th>
+        </tr>
+    </thead>
+    <tbody><tr>
+        <td><b>repository</b></td>
+        <td>string</td>
+        <td>
+          Repository within the Registry where the package is found.<br/>
+        </td>
+        <td>true</td>
+      </tr><tr>
+        <td><b><a href="#packagebundlespecpackagesindexsourceversionsindex">versions</a></b></td>
+        <td>[]object</td>
```

Comment:
Same question about `[]` characters.

Reply:
This shows up a few other times as well.

Reply:
Please see my response in the other question.